flashrom build error

@ioraff 
```
make: ./util/getrevision.sh: No such file or directory
make: ./util/getrevision.sh: No such file or directory
Replacing all version templates with .
Checking for a C compiler... make: ./util/getrevision.sh: No such file or directory
found.
Target arch is x86
Target OS is Linux
Checking for libpci headers... Checking for FTDI support... not found.
Checking if Linux MTD headers are present... found.
Checking version of pci_get_dev... yes.
Checking if Linux SPI headers are present... new version (including PCI domain parameter).
Checking if libpci is present and sufficient... no.
Checking if libz+libpci are present and sufficient...yes.
Checking for utsname support... no.

The following features require libpci: CONFIG_INTERNAL CONFIG_NIC3COM CONFIG_GFXNVIDIA CONFIG_SATASII CONFIG_ATAVIA CONFIG_IT8212 CONFIG_DRKAISER CONFIG_NICREALTEK CONFIG_NICINTEL CONFIG_NICINTEL_SPI CONFIG_NICINTEL_EEPROM CONFIG_OGP_SPI CONFIG_SATAMV.
Please install libpci (package pciutils) and/or libz or disable all features
mentioned above by specifying make CONFIG_ENABLE_LIBPCI_PROGRAMMERS=no
See README for more information.

make: *** [Makefile:1281: hwlibs] Error 1
make: *** Waiting for unfinished jobs....
found.
Checking for clock_gettime support... found.
-> flashrom Build failed
```

--- a/community/pciutils/build
+++ b/community/pciutils/build
@@ -12,4 +12,6 @@ mk() {
 }
 
 mk
-mk DESTDIR="$1" install install-lib
+mk SHARED=yes
+mk DESTDIR="$1" install    install-lib
+mk DESTDIR="$1" SHARED=yes install-lib

